kotsadm/web/src/components/LicenseUploadProgress.jsx

```diff
@@ -52,6 +55,11 @@ class LicenseUploadProgress extends React.Component {
       </div>
     );
 
+    const hasError = this.state.installStatus === "upload_error";
```

Comment:
it's too expensive to have this in `render`, how about moving this to `componentDidUpdate` and only call `onError` when the `installStatus` value changes to `"upload_error"`?